### DISCUSSION TITLE
PhiData Call Agent

### DISCUSSION THREAD

**Discussion by mdwoicke** [agno-agi/agno#1644] [Ideas]
Huge fan of PhiData! It would  be great if PhiData came out with a Call Agent similar to Livekit that uses Realtime 4o model and Twilio.  It is possible to create your own agent and  flow yourself without Livekit but it can get very messy very fast! Would love to collaborate with anyone who would like to add this to PhiData. Thanks!

**Comment by dirkbrnd** [comment]
@mdwoicke yes! We are working on this, so it is on the current roadmap.

Thanks for mentioning!
